src/components/Footer.tsx

```diff
@@ -372,54 +291,92 @@ const Footer: React.FC<IProps> = () => {
         }
       `}
       render={(data) => (
-        <StyledFooter>
-          <FooterTop>
-            <LastUpdated>
+        <Box as="footer" p="1rem 2rem">
+          <Flex
+            fontSize="sm"
+            justify="space-between"
+            alignItems="center"
+            flexWrap="wrap"
+          >
+            <Box color="text200">
               <Translation id="website-last-updated" />:{" "}
               {getLocaleTimestamp(
                 intl.locale as Lang,
                 data.allSiteBuildMetadata.edges[0].node.buildTime
               )}
-            </LastUpdated>
-            <SocialIcons>
-              {socialLinks.map((link, idx) => {
+            </Box>
+            <Box my={4}>
+              {socialLinks.map((link, idk) => {
                 return (
                   <Link
-                    key={idx}
+                    key={idk}
                     to={link.to}
-                    hideArrow={true}
+                    hideArrow
                     color="secondary"
                     aria-label={link.ariaLabel}
                   >
                     <Icon as={link.icon} fontSize="4xl" ml={4} />
                   </Link>
                 )
               })}
-            </SocialIcons>
-          </FooterTop>
-          <LinkGrid>
+            </Box>
+          </Flex>
+          <SimpleGrid
+            gap={4}
+            justifyContent="space-between"
+            gridTemplateColumns="repeat(6, auto)"
+            sx={{
+              "@media (max-width: 1300px)": {
+                gridTemplateColumns: "repeat(3, auto)",
+              },
+              [`@media (max-width: ${smallBp})`]: {
+                gridTemplateColumns: "repeat(2, auto)",
+              },
+              "@media (max-width: 500px)": {
+                gridTemplateColumns: "auto",
+              },
+            }}
```

Comment:
Because some of the breakpoints used are not part of the theme config, there is not a concise way in handling the change of value in Chakra.

`useToken` at line 60 is used to retrieve the correct breakpoint value for `sm` so it can be interpolated here.